inc/views/pluggable/pagination.php

```diff
@@ -128,6 +128,115 @@ public function filter_localization( $data ) {
 		return $data;
 	}
 
+	/**
+	 * Determine whether to show the Jump to page HTML input.
+	 * 
+	 * @return bool
+	 */
+	private function show_jump_to_page_input() {
+		return ( ! $this->has_infinite_scroll() && get_theme_mod( 'neve_enable_jump_to_pagination' ) ) ? true : false;
+	}
+
+	/**
+	 * Create jump to navigation inputs.
+	 * 
+	 * @return mixed $markup HTML to output on page.
+	 */
+	private function create_jump_to_html() {
+
+		// Value escaped once we confirm it's not empty, we never save this value to the DB.
+		$request = $_SERVER['REQUEST_URI'] ?? ''; //phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
@UVLabs Let's try and get the page number that we've submitted via the $_GET parameter and get the $request URL with this function https://developer.wordpress.org/reference/functions/get_pagenum_link/